lib/mayaUsd/utils/util.cpp

```diff
@@ -619,6 +619,18 @@ bool UsdMayaUtil::isRenderable(const MObject& object)
         }
     }
 
+    MDagPath dagPath;
+    stat = mFn.getPath(dagPath);
+    CHECK_MSTATUS_AND_RETURN(stat, false);
+    if (!dagPath.isVisible()) {
+        return false;
+    }
+
+    const MDAGDrawOverrideInfo drawOverrideInfo = dagPath.getDrawOverrideInfo();
+    if (drawOverrideInfo.fOverrideEnabled && drawOverrideInfo.fDisplayType == MDAGDrawOverrideInfo::kDisplayTypeTemplate) {
+        return false;
+    }
```

Comment:
Lgtm! So to be clear, previously this function didn't return the state correctly when the draw override's display type was set to "Template".

![image](https://user-images.githubusercontent.com/48299423/121695380-0190b580-ca99-11eb-83fe-39d87746d416.png)